core/components/atoms/avatarGroup/__stories__/variants/CustomPopper.story.jsx

```diff
@@ -82,7 +79,7 @@ const customCode = `() => {
     });
 
     return (
-      <div className="overflow-auto py-4 px-6" style={{ maxHeight: '150px' }}>
+      <div className="overflow-auto py-4 px-6 Avatar-group">
```

Comment:
Please use some specific class name, `Avatar-group` is too generic and can conflict. The name can be like "UserAvatars-popover".